### PR TITLE
ECM-527 implemented, check styles fixed

### PR DESCRIPTION
**Pull request by gauravchawla73463**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ECM-527

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by hsjhita — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/BulkHelper.java [hunk 1]
```diff
@@ -51,7 +51,8 @@
 public class BulkHelper {
 
     private static final String JURISDICTION_OUTCOME_ACAS_CONCILIATED_SETTLEMENT = "Acas conciliated settlement";
-    private static final String JURISDICTION_OUTCOME_WITHDRAWN_OR_PRIVATE_SETTLEMENT = "Withdrawn or private settlement";
+    private static final String JURISDICTION_OUTCOME_WITHDRAWN_OR_PRIVATE_SETTLEMENT = "Withdrawn or "
+            + "private settlement";
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/BulkHelper.java:55]: Just put "Withdrawn or private settlement" on a new line

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Helper.java [hunk 2]
```diff
@@ -235,6 +238,44 @@ public static void midRespondentECC(CaseData caseData, CaseData originalCaseData
         return errors;
     }
 
+    public static List<String> validateHearingDatesNotInFuture(CaseData caseData) {
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Helper.java:241]: Why did you decide to add code concerned with case validation to a class called Helper?

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Helper.java [hunk 3]
```diff
@@ -235,6 +238,44 @@ public static void midRespondentECC(CaseData caseData, CaseData originalCaseData
...
+        List<String> errors = new ArrayList<>();
+        log.info("Check if dates are not in future for case: " + caseData.getEthosCaseReference());
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Helper.java:243]: Is this log message adding any value?

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Helper.java [hunk 4]
```diff
@@ -235,6 +238,44 @@ public static void midRespondentECC(CaseData caseData, CaseData originalCaseData
...
+        if (CollectionUtils.isEmpty(caseData.getHearingCollection())) {
+            return errors;
+        }
+        for (HearingTypeItem hearingTypeItem : caseData.getHearingCollection()) {
+            if (CollectionUtils.isEmpty(hearingTypeItem.getValue().getHearingDateCollection())) {
+                return errors;
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Helper.java:249]: Is exiting the loop here definitely correct?

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/HelperTest.java [hunk 5]
```diff
@@ -140,6 +141,48 @@ public void hearingMidEventValidationDayError() {
 
     }
 
+    @Test
+    public void validateHearingDatesNotInFutureTest() {
```
Comment (jthmcts) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/HelperTest.java:145]: Would you tests be more readable if you had two methods, one to test for dates in the future and another to test for dates in the past?

**Review by jthmcts — APPROVED** [review]

**Pull request merged**